src/qubex/clifford/clifford_generator.py

```
@@ -0,0 +1,583 @@
+        list_count = len(found_clifford_list)
+        max_gate_count = max(sequence.length for sequence in found_clifford_list)
+
+        print(f"Generated {list_count} unique 1Q1Q Clifford sequences.")
+        print()
+        print(f"  Maximum gate count per Clifford: {max_gate_count}")
+
+        return self._cliffords_1q1q
+
+    def generate_2q_cliffords(
+        self,
+        two_qubit_gate: Clifford = Clifford.ZX90(),
```

Ruff

Do not perform function call `Clifford.ZX90` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)